sdk/purview/azure-analytics-purview-scanning/swagger/README.md

```diff
@@ -21,4 +21,5 @@ add-context-parameter: true
 context-client-method-parameter: true
 generate-sync-async-clients: true
 generate-samples: true
+generate-builder-per-client: false
```

Comment:
scanning seems pretty old. At present add "generate-builder-per-client=false" to keep old behavior.